lightning/src/routing/scoring.rs

```diff
@@ -488,7 +488,7 @@ where L::Target: Logger {
 pub struct ProbabilisticScoringFeeParameters {
 	/// A fixed penalty in msats to apply to each channel.
 	///
-	/// Default value: 500 msat
+	/// Default value: 1000 msat
```

Comment:
@TheBlueMatt  

Should we use this opportunity to define constants for each parameter?
This could simplify documentation maintenance and allow us to define constants cleanly using multiplier syntax.  

Something like:

```rust
const BASE_PENALTY_MSAT: u64 = 100 * 1000;
const BASE_PENALTY_AMOUNT_MULTIPLIER_MSAT: u64 = 8192 * 100;
...

Reply:
Ben showed the operands to show how much mutiny deviated from the defaults - I think we should set a single number for the default settings.